deploy_scripts/install.sh

```diff
@@ -1,31 +1,5 @@
 #!/bin/bash
-# shellcheck disable=SC1090,SC1091
 
 set -euo pipefail
 
-set +u
-source /home/ec2-user/.bash_profile
-set -u
-
-# section: pre_install_flashcrow
-## /install_node.sh
-if command -v nvm > /dev/null 2>&1; then
-  echo "nvm already installed, skipping..."
-else
-  echo "installing nvm..."
-  curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.36.0/install.sh | bash
-  # ensure that nvm shims are available in current shell session
-  export NVM_DIR="$HOME/.nvm"
-  [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
-  [ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"  # This loads nvm bash_completion
-fi
-
-# confirm
-echo "$NVM_DIR"
-
-# install correct version of node
-echo "installing node@lts/*..."
-nvm install lts/*
-echo "lts/*" > /home/ec2-user/.nvmrc
-nvm use lts/*
 npm install -g forever
```

Comment:
this won't work, as there's no global `node` to call - we'd need the `PATH` that's set in `start.sh` to make this work.

instead of doing that, though, let's just move this down to `start.sh` - it's a bit strange in that we're then installing dependencies in the start script, but it does allow us to avoid the complexity of setting this `PATH` in two separate scripts.